### ISSUE TITLE
Failed create commit status in docker container

### ISSUE THREAD

**Issue by duck8823** [duck8823/duci#122]
If running server in docker container, failed create commit status and clone with http.

```
x509: certificate signed by unknown authority
```

**Comment by theoden9014** [comment]
Would you update CA certificates?
Try to add the following line to `Dockerfile`.
```
apk add --update --no-cache ca-certificates && update-ca-certificates
```

**Comment by duck8823** [comment]
@theoden9014 
thanks your advice!
I fixed this bug in #124 and released `v0.6.2` .
Now, available as `latest` in dockerhub https://hub.docker.com/r/duck8823/duci/tags/
